### ISSUE TITLE
(1.12.2) I know you dont support it anymore - AppliedEnergistics2 incompatibility? 

### ISSUE THREAD

**Issue by Erosion139** [ErdbeerbaerLP/DiscordIntegration-Forge#388]
### What happened?

My large modpack needed an alternative to another discord integration so I tried out this one, seemed lively. 

Unfortunately, I am producing a crash when this mod is installed and it appears to have something to do with AE2.

### Game Version

1.12

### What platform are you using?

Forge

### Platform version

14.23.5.2860

### Config File

```toml
Unable to produce config file.
```

### Crash report / Error log (if applicable)

```shell
https://gist.github.com/Erosion139/64c0fc1151825d41f52bab6b505549aa
```

**Comment by ErdbeerbaerLP** [comment]
No, that's just an command registration issue. Check bot permissions, maybe reinvite with url from guide
